Tests/AK/TestSIMDExtras.cpp

```diff
@@ -0,0 +1,1399 @@
+/*
+ * Copyright (c) 2024,  Marek Knápek <knapek.mar@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <AK/SIMD.h>
+#include <AK/SIMDExtras.h>
+
+// See the comment in <AK/SIMDMath.h>
+#pragma GCC diagnostic ignored "-Wpsabi"
```

Comment:
Maybe just disable it for lagom and userland? I'm not sure if this warning has ever warned about something useful.